src/combinations.rs

```diff
@@ -143,6 +143,27 @@ impl<I: Iterator> Combinations<I> {
         // If we've made it this far, we haven't run out of combos
         false
     }
+
+    /// Returns the n-th item or the remaining steps.
```

Comment:
I may be biased, but I feel it would be simpler to return the number of successful steps (similar to how e.g. [`Read::read`](https://doc.rust-lang.org/std/io/trait.Read.html#tymethod.read) returns how many bytes were already read).

Reply:
I was not sure, I sure don't mind change this.